src/main/java/Domain/UserController.java

```diff
@@ -34,7 +35,7 @@ public boolean isCredentialsCorrect(User user, String password) {
         return this.passwordEncoder.matches(password, user.getEncodedPassword());
     }
 
-    public void register(String user_name, String password, String email) throws Exception{
+    public void register(String user_name, String password, String email) throws Exception {
```

Comment:
documents and logs pls

Reply:
This class is not my responsibility, read my commit messages, I have an auto formatter that just fixed spacing